### ISSUE TITLE
THANK YOU

### ISSUE THREAD

**Issue by Eliav2** [mdx-editor/editor#304]
This project is one of the best React markdown editor projects I've seen. Just well written, fully customizable, and well documented! I just as open source contributor, I know that sometimes a project can feel unappreciated, so I just wanted to say thank you for this awesome project!

this issue can be safely closed 😁

**Comment by dharanish-v** [comment]
Yes, absolutely with no doubt, This is one of The BEST REACT MARKDOWN EDITORS!!!
